tests_new/integration_tests/modules/s3_datasets/test_s3_dataset.py

```diff
@@ -284,3 +285,14 @@ def test_persistent_import_kms_s3_dataset_update(client1, persistent_imported_km
         client1, env_uri=env_uri, stack_uri=stack_uri, target_uri=dataset_uri, target_type=target_type
     )
     assert_that(stack.status).is_in('CREATE_COMPLETE', 'UPDATE_COMPLETE')
+
+
+def test_list_object_keys(client1, persistent_s3_dataset1):
+    response = list_object_keys(client1, persistent_s3_dataset1.datasetUri)
+    assert_that(response).is_empty()
```

Comment:
Is it always going to be empty?

Reply:
yes... until someone changes other tests to start uploading pdfs and txt files to the persistent_s3_dataset1 S3 Bucket 😄

Reply:
The time will come :)